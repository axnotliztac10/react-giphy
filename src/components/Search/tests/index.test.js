import React from 'react';
import toJSON from 'enzyme-to-json';
import { shallow } from 'enzyme';

import { Search } from '../index';

describe('<Search />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Search
        handleSubmit={() => {}}
        onSearch={() => {}}
      />
    );

    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
