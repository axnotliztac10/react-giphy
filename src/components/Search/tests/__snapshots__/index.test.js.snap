// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> should render correctly 1`] = `
<Card
  my={4}
  p={2}
  px={3}
>
  <form>
    <styled.label>
      Search: 
    </styled.label>
    <Field
      component={[Function]}
      name="search"
    />
    <Styled(styled.div)
      as="button"
      bg="magenta"
      border={0}
      borderRadius={4}
      color="white"
      fontSize="inherit"
      fontWeight="bold"
      m={0}
      px={3}
      py={2}
    >
      Go
    </Styled(styled.div)>
  </form>
</Card>
`;
